Makefile

```diff
@@ -20,11 +20,11 @@
 #==================================================================================================
 
 
-CPP                =
+CPP                = 
 PYTHON             = python
 COMPILER_MODE      = DEBUG
 PRECISION          = DOUBLE
-OPENMP             = 1
+OPENMP             = 0
```

Comment:
Ok, I'd forgotten to add the libmpi++ which we seem to need. 


I would suggest that we discourage people from setting MPI=1 themselves since the mpic++ compiler does a much better job of finding the necessary headers/libraries. For example on my laptop I need to specify the location of the MPI header. I'd suggest that by default we don't even list the MPI=0 line, but state in the documentation how to go about doing it if you don't have the MPI wrappers. I.e.:
1) set MPI=1 in the makefile
2) If compilation fails you may need to:
 a) set INCLUDE to point to the directory containing mpi.h
 b) modify the link options, -lmpi -lmpi++ to reflect the library installed.